assets/javascripts/discourse/routes/activity-pub-actor.js

```diff
@@ -57,4 +57,16 @@ export default DiscourseRoute.extend({
       }
     );
   },
+
+  @action
+  reject(actor, followingActor) {
+    return ActivityPubActor.reject(actor.id, followingActor.id).then(
```

Comment:
Could probably use `async/await` here.

Reply:
We could. We'd need to change the code that relies on this action too. Is async/await the preferred approach now?